### PR TITLE
Internationalisation Support

### LINKED ISSUE #10: Internationalization support
Currently, this plugin has 0 translation support.

### PR DESCRIPTION
**Pull request by drfiveminusmint**
Adds internationalisation support to all messages. Tested on 1.12.2, but I see no reason it wouldn't work on any other version. Currently the only language is English.

This will close #10

### CODE REVIEW

File: src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/AADirectorSign.java [hunk 1]
```diff
@@ -45,24 +46,24 @@ public final void onSignClick(PlayerInteractEvent event) {
             }
         }
         if (foundCraft == null) {
-            player.sendMessage(ERROR_PREFIX + "Sign must be part of a piloted craft!");
+            player.sendMessage(ERROR_PREFIX + I18nSupport.getInternationalisedString("Sign - Must Be Part Of Craft"));
```
Comment (TylerS1066) [inline src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/AADirectorSign.java:49]: This needs a space to display correctly.

File: src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/AADirectorSign.java [hunk 2]
```diff
@@ -45,24 +46,24 @@ public final void onSignClick(PlayerInteractEvent event) {
...
             return;
         }
 
         if (!Config.AADirectorsAllowed.contains(foundCraft.getType())) {
-            player.sendMessage(ERROR_PREFIX + "AA Director signs not allowed on this craft!");
+            player.sendMessage(ERROR_PREFIX + I18nSupport.getInternationalisedString("AADirector - Not Allowed On Craft"));
```
Comment (TylerS1066) [inline src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/AADirectorSign.java:54]: This needs a space to display correctly.

File: src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/CannonDirectorSign.java [hunk 3]
```diff
@@ -45,24 +46,24 @@ public final void onSignClick(PlayerInteractEvent event) {
             }
         }
         if (foundCraft == null) {
-            player.sendMessage(ERROR_PREFIX + "Sign must be part of a piloted craft!");
+            player.sendMessage(ERROR_PREFIX + I18nSupport.getInternationalisedString("Sign - Must Be Part Of Craft"));
```
Comment (TylerS1066) [inline src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/CannonDirectorSign.java:49]: This needs a space to display correctly.

File: src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/CannonDirectorSign.java [hunk 4]
```diff
@@ -45,24 +46,24 @@ public final void onSignClick(PlayerInteractEvent event) {
...
             return;
         }
 
         if (!Config.CannonDirectorsAllowed.contains(foundCraft.getType())) {
-            player.sendMessage(ERROR_PREFIX + "Cannon Director signs not allowed on this craft!");
+            player.sendMessage(ERROR_PREFIX + I18nSupport.getInternationalisedString("CannonDirector - Not Allowed On Craft"));
```
Comment (TylerS1066) [inline src/main/java/net/countercraft/movecraft/combat/movecraftcombat/sign/CannonDirectorSign.java:54]: This needs a space to display correctly.

File: src/main/java/net/countercraft/movecraft/combat/movecraftcombat/status/StatusManager.java [hunk 5]
```diff
@@ -132,14 +133,14 @@ private void startCombat(@NotNull Player player) {
         if(isInCombat(player))
             return;
         Bukkit.getServer().getPluginManager().callEvent(new CombatStartEvent(player));
-        player.sendMessage(ChatColor.RED + "You have now entered combat.");
-        MovecraftCombat.getInstance().getLogger().info(player.getName() + " has left combat.");
+        player.sendMessage(ChatColor.RED + I18nSupport.getInternationalisedString("Status - Enter Combat"));
+        MovecraftCombat.getInstance().getLogger().info(player.getName() + I18nSupport.getInternationalisedString("Log - Enter Combat"));
```
Comment (TylerS1066) [inline src/main/java/net/countercraft/movecraft/combat/movecraftcombat/status/StatusManager.java:137]: This needs a space to display correctly.

File: src/main/java/net/countercraft/movecraft/combat/movecraftcombat/status/StatusManager.java [hunk 6]
```diff
@@ -132,14 +133,14 @@ private void startCombat(@NotNull Player player) {
...
     }
 
     private void stopCombat(@NotNull Player player) {
         Bukkit.getServer().getPluginManager().callEvent(new CombatStopEvent(player));
-        player.sendMessage(ChatColor.RED + "You have now left combat.");
-        MovecraftCombat.getInstance().getLogger().info(player.getName() + " has left combat.");
+        player.sendMessage(ChatColor.RED + I18nSupport.getInternationalisedString("Status - Leave Combat"));
+        MovecraftCombat.getInstance().getLogger().info(player.getName() + I18nSupport.getInternationalisedString("Log - Leave Combat"));
```
Comment (TylerS1066) [inline src/main/java/net/countercraft/movecraft/combat/movecraftcombat/status/StatusManager.java:143]: This needs a space to display correctly.

**Comment by drfiveminusmint** [comment]
Requested changes made @TylerS1066 (assuming I didn't misinterpret what you wanted me to do)

**Pull request merged**